examples/check_examples

```diff
@@ -47,6 +47,50 @@ set -e
 export PSYCLONE=${PWD}/../bin/psyclone
 export KSTUBGEN=${PWD}/../bin/genkernelstub
 
+# Ensure we use the configuration file distributed with PSyclone and
+# not a (potentially modified) local version.
+export PSYCLONE_CONFIG=${PWD}/../config/psyclone.cfg
```

Comment:
Perhaps report to the screen that we're doing this as someone might try this and expect their config file to be used.

Reply:
Not sure if this helps with part of the issue: this allows a user to overwrite the setting (but echoing the value should probably still be done, but from memory the output of the script is huge.
```
export PSYCLONE_CONFIG=${PSYCLONE_CONFIG:-${PWD}/../config/psyclone.cfg}
```

Reply:
I think we should stick with how I've implemented it - we ensure that the test suite is insulated from any user-modified config file (well, unless they modify the one in `config/` but then that's just tough) and I think we should do the same with the examples. Anything else is asking for confusion. I'll add a report to the screen so at least a user can find it, even though, as @hiker says, it will be swamped by the rest of the output.

Reply:
Redirect the output of all scripts we call to stderr? Then we can print a short summary to stdout (git branch, psyclone version(?), config file used).

Reply:
Good idea but I've punted it to #713.

Reply:
I'm happy with the punt.